### DISCUSSION TITLE
mpv的osc有办法在视频之外显示吗？

### DISCUSSION THREAD

**Discussion by htyxyt** [hooke007/mpv_PlayKit#492] [Q&A 问答]
不论是官方osc还是modernx，都是一个在视频底部半透明的界面。虽然是半透明的，但还是有点遮挡。有没有办法像mpc、vlc那种一样，让控制器显示在视频之外，完全不遮挡？

**Comment by hooke007** [comment]
1.你猜它为什么叫osc/osd而不是真的UI
2.其它播放器的osc也是局限在输出区域的（参见madvr exclusive模式下显示的seek bar）
3.如果你能接受渲染在黑边，那也勉强能算

**Reply by htyxyt** [reply]
> 1.你猜它为什么叫osc/osd而不是真的UI 2.其它播放器的osc也是局限在输出区域的（参见madvr exclusive模式下显示的seek bar） 3.如果你能接受渲染在黑边，那也勉强能算
谢谢，明白了！
不过这到也是个思路诶，那么有没有办法让视频的底部自动添加一定宽度的黑边呢？

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#options-video-margin-ratio-bottom

**Reply by htyxyt** [reply]
> https://mpv.io/manual/master/#options-video-margin-ratio-bottom

谢谢，我来试试！
